libpod/runtime_volume_common.go

```diff
@@ -201,7 +201,7 @@ func (r *Runtime) newVolume(ctx context.Context, noCreatePluginVolume bool, opti
 				Inodes: volume.config.Inodes,
 				Size:   volume.config.Size,
 			}
-			if err := q.SetQuota(fullVolPath, quota); err != nil {
+			if err := q.SetQuota(volPathRoot, quota); err != nil {
```

Comment:
Can you add some of the commit message as comment here, as soon as the code gets moved it will be harder to find the reason why this must be the root.

Reply:
Done